contracts/test_argent_account_escape.cairo

```diff
@@ -0,0 +1,111 @@
+use contracts::ArgentAccount;
+use contracts::dummy_syscalls;
+
+const INITIALIZED_SIGNER: felt = 1;
+const INITIALIZED_GUARDIAN: felt = 2;
+
+#[test]
+#[available_gas(2000000)]
+fn valid_trigger_escape_signer() {
+    ArgentAccount::initialize(INITIALIZED_SIGNER, INITIALIZED_GUARDIAN, 0);
+    ArgentAccount::trigger_escape_signer();
+
+    let escape_active_at = ArgentAccount::get_escape_active_at();
+    let escape_type = ArgentAccount::get_escape_type();
+    let block_timestamp = dummy_syscalls::get_block_timestamp();
+    assert(
+        escape_active_at == block_timestamp + ArgentAccount::ESCAPE_SECURITY_PERIOD,
+        'escape activation set to wrong timestamp'
+    );
+    assert(escape_type == ArgentAccount::ESCAPE_TYPE_SIGNER, 'wrong escape type');
+}
+
+#[test]
+#[available_gas(2000000)]
+#[should_panic(expected = 'argent: cannot override escape')]
+fn trigger_escape_signer_escape_already_active() {
```

Comment:
this test should not panic. keep it but let it pass.

then:
- add the guardian escape methods
- add a panicking test where the signer triggers a guardian escape, and then the guardian unsuccessfully tries a signer escape
- add a non-panicking test where the guardian triggers a signer escape, and the the signer successfully replaces the escape by a guardian escape

the reason is that the signer always has priority over the guardian for censorship resistance (because guardian could become compromised etc)